sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_gen2_storage_helper.py

```diff
@@ -6,35 +6,23 @@
 import logging
```

Comment:
I can't leave this comment on the image file itself - but why is there an PNG file in the SDK?
Is this somehow public? If so - how? If not, it should be removed, it's adding ~100kb to the package.

Reply:
You're right, this is better served as a doc outside of the source code. It's gone now.